test/e2e-olm/elasticsearch_test.go

```diff
@@ -176,75 +319,91 @@ func elasticsearchFullClusterTest(t *testing.T, f *framework.Framework, ctx *fra
 	//	return fmt.Errorf("Timed out waiting for full cluster restart to complete")
 	//}
 
-	// ensure all prior nodes are ready again
-	err = e2eutil.WaitForDeployment(t, f.KubeClient, namespace, fmt.Sprintf("elasticsearch-cdm-%v-1", dataUUID), 1, retryInterval, timeout)
+	t.Log("Waiting for redeployment after secret update")
```

Comment:
just a side comment, but my concern with this test (as with the original one) is we don't know that the pods have actually been redeployed or if we just move on because they're still in the same state as before.

Reply:
We decided not to wait for pods here.